seqkit/cmd/sort.go

```diff
@@ -63,14 +63,14 @@ and extract sequences by FASTA index.
 		// lineWidth := config.LineWidth
 		outFile := config.OutFile
 		quiet := config.Quiet
-		seq.AlphabetGuessSeqLengthThreshold = config.AlphabetGuessSeqLength
+		//seq.AlphabetGuessSeqLengthThreshold = config.AlphabetGuessSeqLength
 		seq.ValidateSeq = false
 		fai.MapWholeFile = false
 		runtime.GOMAXPROCS(config.Threads)
 
 		files := getFileList(args)
 
-		inNaturalOrder := getFlagBool(cmd, "natural-order")
+		//inNaturalOrder := getFlagBool(cmd, "natural-order")
```

Comment:
The reason?

Reply:
didn't compile. don't yet understand root cause. reproducible on your end?